migrations/1617742522808-zeroConfLimit-migrate.js

```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash/fp')
+var db = require('../lib/db')
+const settingsLoader = require('../lib/new-settings-loader')
+const configManager = require('../lib/new-config-manager')
+
+const isNil = val => val == null
```

Comment:
As I pointed on another review, you can call `_.isNil()` or `_.isNull()`, instead of creating a new function for it